pages/common/huggingface-cli.md

```diff
@@ -0,0 +1,37 @@
+# huggingface-cli
+
+> Interact with Huggingface Hub.
```

Comment:
```suggestion
> Interact with Hugging Face Hub.
```
Like in their documentation.

![IMG_20240704_153641.jpg](https://github.com/tldr-pages/tldr/assets/155078792/38e50b68-4616-42eb-923e-0444f1c23d8b)

Reply:
Fixed this with the latest commit